pkg/local.go

```diff
@@ -41,7 +42,7 @@ func (p *LocalPackage) Install(ctx context.Context, name, dir, version string) (
 	}
 
 	oldname := filepath.Join(wd, p.Source.Directory)
-	newname := filepath.Join(wd, filepath.Join(dir, name))
+	newname := filepath.Join(dir, name)
```

Comment:
What about renaming `dir` to `vendorDir` ?

Reply:
Hmmm I think "vendor" would reflects the default value more than the purpose.
The official name for this config param is `jsonnetpkg-home` / `JsonnetHome`:

```go
	a.Flag("jsonnetpkg-home", "The directory used to cache packages in.").
		Default("vendor").StringVar(&cfg.JsonnetHome)
```

The code base is pretty loose on this... almost everywhere it's just called `dir`.